### DISCUSSION TITLE
new tables in database

### DISCUSSION THREAD

**Discussion by MiroslavHrcan** [developmentseed/timvt#93] [General] [answered]
Is there some nice way to reconnect to database when TiMVT is deployed as docker container? When I restart container, TiMVT sees new tables, but as I am planning to deploy it on AWS via ECS and don't want to re-visioning task each time I add a data into database.
Best regards,
Miroslav

**Comment by vincentsarago** [comment, marked as answer]
In theory you could add an endpoint to run the `table registration` 

```python
@app.get("/register", include_in_schema=False)
async def register_table(request: Request):
    await register_table_catalog(request.app)
```
see https://github.com/developmentseed/tifeatures/issues/49#issuecomment-1205331979

Or you could add a middleware that call `register_table_catalog` but this might slow down the application.

**Reply by MiroslavHrcan** [reply]
Okay, I will try with this endpoint solution.
Thank you for the answer.
